lib/src/commons/BrowserUtils.d.ts

```diff
@@ -151,7 +151,7 @@ export declare namespace BrowserUtils {
      * Get text of an element by selector
      * @param selector element's selector
      */
-    function getText(selector: string): string;
+    function getText(selector: string): string[];
```

Comment:
should return single string value or throw a meaningful error if more then one found